build.sbt

```diff
@@ -171,7 +171,11 @@ lazy val js = project.in(file("js"))
 lazy val jvm = project.in(file("jvm"))
   .settings(sharedSettings: _*)
   .settings(
-    fork in Test := true,
+    fork in Test := {
+      // Serialization issue in 2.13
+      scalaMajorVersion.value == 13 // ==> true
+      // else ==> false
+    },
```

Comment:
Is there a reason we don't want to fork unconditionally?

Reply:
I'm not sure.  While working on fixing the 2.13 serialization issue, this was changed to forking unconditionally for all versions.  I assumed it was a change made broadly in the narrow context of needing to publish 2.13.  I'm just trying to revert back.

It would likely be useful to get coverage for both cases, but that may be more growth (a doubling!) to the CI build than we'd want to entertain.

Reply:
OK, sure, that works.